torch_geometric/explain/explanation.py

```diff
@@ -245,6 +245,53 @@ def visualize_feature_importance(
 
         return ax
 
+    def visualize_subgraph(self, node_index: int) -> str:
+        r"""Threshold the explanation mask according to the thresholding method.
+        Generate and explanation subgraph visualization for thresholded and non-thresholded explanation.
+        The output would be a visualization of the explanation subgraph where edge width depends on the importance value of each edge.
+        Args:
+            explanation (Explanation): The explanation to visualize.
+        """
+        import graphviz
+
```

Comment:
```suggestion
    assert getattr('edge_mask', self, None) is not None
```
All `Explnation`s might not have an `edge_mask`.